### ISSUE TITLE
How many epochs will it take to get a good result?

### ISSUE THREAD

**Issue by Happzy-WHU** [LambdaLabsML/examples#32]
I also have a question about the accuracy of the training time. I trained 24h with 8xA40, but only 150 epochs.

**Comment by pfdamasceno** [comment]
I've got good results with < 100 epochs. More than that and the images actually get worse as mentioned in the blog.

**Comment by Yeez-lee** [comment]
> I've got good results with < 100 epochs for the fine-tuning example. More than that and the images actually get worse as mentioned in the blog.

Hi, do you know how to set epoch? I didn't find it.
